cli/pack/deb.go

```diff
@@ -67,65 +67,64 @@ func (packer *debPacker) Run(cmdCtx *cmdcontext.CmdCtx) error {
 	}
 	defer os.RemoveAll(bundlePath)
 
+	log.Infof("Creating a data directory")
+
 	// Create a data directory.
 	rootPrefixDir := dataDirName
 	rootPrefix := filepath.Join(rootPrefixDir, debBundlePath)
-	if packCtx.Name != "" {
-		rootPrefix = fmt.Sprintf(rootPrefix, packCtx.Name)
-	} else {
-		rootPrefix = fmt.Sprintf(rootPrefix, "bundle")
+
+	packageName, err := getPackageName(&cmdCtx.Pack, "", false)
```

Comment:
Why is there not .deb suffix?

Reply:
I need a non suffixed name here for creating a directory in /usr/share/tarantool with this name, where the installed package will be located.